scripts/paas_app_wrapper.sh

```diff
@@ -33,7 +33,7 @@ case $NOTIFY_APP_NAME in
     -Q reporting-tasks 2> /dev/null
     ;;
   delivery-worker-priority)
-    exec scripts/run_app_paas.sh celery -A run_celery.notify_celery worker --loglevel=INFO --concurrency=5 \
+    exec scripts/run_app_paas.sh celery -A run_celery.notify_celery worker --loglevel=INFO --concurrency=4 \
```

Comment:
Quick drive by comment. There are several apps here running with concurrency not equal to 4. It would be good I think to either
- make it consistent
- detail in the commit message which ones are not being set to 4 and why please

Reply:
The lower concurrency tasks don't need to change at this point as they appear to be fine with long polling enabled. So, in the interests of changing one thing at a time, I feel it's okay to leave them as they are. The high-concurrency tasks were definitely running too many processes with long-polling enabled. I expect we may need to tweak these more scientifically, both for performance and economy.

Reply:
Made it consistent in https://github.com/alphagov/notifications-api/pull/3355/commits/9e9091e980c6c2a10a47cc7c443ecadbcaf29512